deepmd/descriptor/se_a.py

```diff
@@ -956,7 +956,7 @@ def _filter_lower(
                     extra_embedding_index = self.nei_type_vec
                 else:
                     padding_ntypes = type_embedding.shape[0]
-                    atype_expand = tf.reshape(self.atype, [-1, 1])
+                    atype_expand = tf.reshape(self.atype_nloc, [-1, 1])
                     idx_i = tf.tile(atype_expand * padding_ntypes, [1, self.nnei])
                     idx_j = tf.reshape(self.nei_type_vec, [-1, self.nnei])
                     idx = idx_i + idx_j
```

Comment:
@nahso It seems this block of codes is never used?

Reply:
It is used in line 1061 below, which provide the index of `net_output`.

Reply:
I see!